src/main/java/com/jmb/springfactory/service/UtilsService.java

```diff
@@ -24,6 +31,24 @@ public static void throwNotFoundExceptionIfEmptyList(List<?> list) throws NotFou
       throw new NotFoundException();
     }
   }
+
+  /**
+   * Add a new element into the list param if this exists, if not exists, create a new list and append the new element
+   * @param orders
+   * @param order
+   * @return
+   */
+  public static List<?> addIntoList(List<?> orders, Object order) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove usage of generic wildcard type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1452)